src/router_sc_worker.erl

```diff
@@ -326,13 +333,23 @@ get_nonce(PubkeyBin, Ledger) ->
 
 -spec active_sc_expiration() -> {error, no_active_sc} | {ok, pos_integer()}.
 active_sc_expiration() ->
-    case blockchain_state_channels_server:active_sc_id() of
-        undefined ->
+    case blockchain_state_channels_server:active_sc_ids() of
```

Comment:
Can we take a min of all the expirations here, and not have to deal with the potential of the maps blowing up?
```
active_sc_expiration() ->
    case blockchain_state_channels_server:active_scs() of
        [] ->
            {error, no_active_sc};
        ActiveSCs ->
            Expires = [blockchain_state_channel_v1:expire_at_block(SC) || SC <- ActiveSCs],
            {ok, lists:min(Expires)}
    end.
```